docs/io/visualization/interaction_radius_plot.ipynb

```diff
@@ -0,0 +1,7286 @@
+{
```

Comment:
<ol><li>Fix heading level here too</li><li>"interaction radius plot" --&gt; "last interaction velocity plot"</li></ol><p> </p><p>Also, I think it is more important to show logscale plot than range clipped - so add a section for that (and feel free to keep this or remove it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>